src/fontra/views/editor/panel-glyph-search.js

```diff
@@ -37,4 +83,13 @@ export default class GlyphSearchPanel extends Panel {
   }
 }
 
+export function glyphInfoFromGlyphName(glyphName, fontController) {
```

Comment:
Let's make this function a method of FontController instead. I don't like how editor needs to import this from a panel module.